src/app/components/label-selector/label-selector.component.html

```diff
@@ -13,7 +13,7 @@
   <ng-template #toggleButtonRef>
     <span class="clickable add-label" (click)="closeAddLabel()">
       <i class="pficon pficon-add-circle-o"></i>
-      Add label
+      Assign labels
```

Comment:
@Preeticp is it Assign label or Assign Label?

Reply:
Ideally, I would suggest Add label, but we have Title case being used in majority of the cases in OSIO, so to keep it consistent let's use Assign Label.

Reply:
If it has to be changed it should be across OSIO as a whole to sentence case.

Reply:
Thanks @Preeticp 
@debloper please make it "Assign Label"

Reply:
Please reflect whether it should be `Label` or `Labels` cause it deals with multiple association.

Reply:
@Preeticp So should it be "Assign Labels" ? Can you confirm.